sdk/src/connections/target/rgb_sensor.cpp

```diff
@@ -432,10 +432,9 @@ aditof::Status RgbSensor::getFrame(uint16_t *buffer,
             return status;
         }
 
-        int k = 0;
-        for (unsigned int j = 0; j < buf_data_len; j = j + 2) {
-            buffer[k++] = (pdata[i][j + 1] << 8) + pdata[i][j];
-        }
+        //conversion done on the target
+        RgbSensor::bayer2RGB(buffer, pdata[i], RGB_FRAME_WIDTH,
```

Comment:
not a real problem but I don't think "RgbSensor::" adds value. We know bayerRGB is a method of this class. I don't think it helps explicitly saying so.